kubeflow/core/jupyterhub_spawner.py

```diff
@@ -91,3 +91,18 @@ def extra_resource_limits(self):
 # How much disk space do we want?
 c.KubeSpawner.user_storage_capacity = '10Gi'
 c.KubeSpawner.pvc_name_template = 'claim-{username}{servername}'
+c.KubeSpawner.volumes = [
+  {
+    'name': 'volume-{username}{servername}',
+    'persistentVolumeClaim': {
+      'claimName': 'claim-{username}{servername}'
+    }
+  }
+]
+c.KubeSpawner.volume_mounts = [
+  {
+    'mountPath': '/home/jovyan/work',
```

Comment:
Should everything in /home be on the PVC?

Reply:
well we have tf examples there too, which I think is a nice touch, and that would disappear.

Reply:
What about about putting it under, /mnt ? I'd like to make it more obvious which directories correspond to PD and which don't.

Reply:
jupyter tree lands in /home/jovyan, so if we put it in /mnt it won't be obvious how to use it by users.

Reply:
Ok. Please address other comments.